src/NCRings.jl

```diff
@@ -176,6 +176,14 @@ function is_nilpotent(a::T) where {T <: NCRingElement}
 end
 
 
+@doc raw"""
+    is_square(a::T)  where {T <: NCRingElement}
```

Comment:
```suggestion
    is_square(a::NCRingElement)
```
Of course this is a very minor complaint. But at least to me these function signatures are confusing, I always start wondering if there is a reason to write it with the type parameter. And I don't see a reason to write it this way in a docstring.